samplecode/src/main/java/com/microsoft/azure/sample/User.java

```diff
@@ -9,10 +9,13 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.microsoft.azure.spring.data.documentdb.core.mapping.Document;
+import com.microsoft.azure.spring.data.documentdb.core.mapping.DocumentIndexingPolicy;
 import com.microsoft.azure.spring.data.documentdb.core.mapping.PartitionKey;
+import com.microsoft.azure.documentdb.IndexingMode;
 import org.springframework.data.annotation.Id;
 
 @Document(collection = "mycollection")
+@DocumentIndexingPolicy(mode = IndexingMode.Lazy, automatic = false, includePaths = {}, excludePaths = {})
```

Comment:
also add some comments in readme file to explain this line?

Reply:
The README has covered this part clearly. So I consider it is unnecessary here.